### ISSUE TITLE
✅ 🐞 Bug Report: Post Excerpt in new Grid Loop not displaying anything - [ED-10128]

### ISSUE THREAD

**Issue by CharfishCharlie** [elementor/elementor#20256]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Using the new Grid Loop module, the Post Excerpt module displays nothing. Post Title and Featured Image work fine, but Post Excerpt never shows any content.

### Steps to reproduce

1. Load up latest Elementor, Elementor Pro, and Beta plugins
2. Create a new page and bring in Grid Loop module
3. Edit the loop template
4. Drag in the Title, Featured Image, and Post Excerpt modules to create a nice loop
5. Post Excerpt displays nothing in preview and nothing when viewing the saved page

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Darwin
	Software: nginx/1.16.0
	MySQL version: MySQL Community Server - GPL v8.0.16
	PHP Version: 8.0.0
	PHP Memory Limit: 256M
	PHP Max Input Vars: 4000
	PHP Max Post Size: 1000M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1
	Site URL: http://localhost:10038
	Home URL: http://localhost:10038
	WP Multisite: No
	Max Upload Size: 300 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: Plain
	Language: en-US
	Timezone: 0
	Admin Email: capabst@gmail.com
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.6.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.9.0-dev4
		Author: Elementor.com

	Elementor Beta (Developer Edition)
		Version: 1.1.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.8.0
		Author: Elementor.com


== Elements Usage ==
	
	wp-page : 6
		button : 9
		divider : 9
		google_maps : 1
		heading : 45
		icon-box : 31
		icon-list : 5
		image : 20
		image-box : 21
		image-carousel : 1
		loop-grid : 1
		star-rating : 3
		testimonial : 1
		text-editor : 13
	loop-item : 1
		container : 1
		theme-post-excerpt : 2
		theme-post-featured-image : 1
		theme-post-title : 1


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Additional Custom Breakpoints: Active by default
	Import Export Website Kit: Active by default
	Hide native WordPress widgets from search results: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active by default
	Landing Pages: Active by default
	Color Sampler: Active by default
	Favorite Widgets: Active by default
	Admin Top Bar: Active by default
	Lazy Load Background Images: Active by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default


== Log ==
	
Log: showing 20 of 232022-05-31 14:26:05 [info] Elementor/Upgrades - _on_each_version Finished 
2022-05-31 14:26:05 [info] Elementor/Upgrades - _v_3_2_0_migrate_breakpoints_to_new_system Start  
2022-05-31 14:26:05 [info] Elementor/Upgrades - _v_3_2_0_migrate_breakpoints_to_new_system Finished 
2022-05-31 14:26:05 [info] Elementor/Upgrades - _v_3_4_8_fix_font_awesome_default_value_from_1_to_yes Start  
2022-05-31 14:26:05 [info] Elementor/Upgrades - _v_3_4_8_fix_font_awesome_default_value_from_1_to_yes Finished 
2022-05-31 14:26:05 [info] Elementor/Upgrades - _v_3_5_0_remove_old_elementor_scheme Start  
2022-05-31 14:26:05 [info] Elementor/Upgrades - _v_3_5_0_remove_old_elementor_scheme Finished 
2022-05-31 14:26:05 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.1.1',
  'to' => '3.6.5',
)]
2022-11-02 20:33:45 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.6.5',
  'to' => '3.8.0',
)]
2022-11-02 20:33:45 [info] elementor::elementor_updater Started 
2022-11-02 20:33:45 [info] Elementor/Upgrades - _on_each_version Start  
2022-11-02 20:33:45 [info] Elementor/Upgrades - _on_each_version Finished 
2022-11-02 20:33:45 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Start  
2022-11-02 20:33:45 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Finished 
2022-11-02 20:33:45 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.6.5',
  'to' => '3.8.0',
)]
2022-11-02 20:38:30 [info] elementor::elementor_updater Started 
2022-11-02 20:38:30 [info] Elementor/Upgrades - _on_each_version Start  
2022-11-02 20:38:30 [info] Elementor/Upgrades - _on_each_version Finished 
2022-11-02 20:38:30 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.0',
  'to' => '3.9.0-dev4',
)]
2022-11-02 20:38:30 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.0',
  'to' => '3.9.0-dev4',
)]

PHP: showing 2 of 2PHP: 2022-11-02 20:34:10 [error X 2][/Users/parley/Local Sites/create-a-blog-with-elementor/app/public/wp-content/plugins/elementor-pro/modules/global-widget/module.php::32] Uncaught Error: Call to undefined method Elementor\Core\Editor\Editor::add_editor_template() in /Users/parley/Local Sites/create-a-blog-with-elementor/app/public/wp-content/plugins/elementor-pro/modules/global-widget/module.php:32
Stack trace:
#0 /Users/parley/Local Sites/create-a-blog-with-elementor/app/public/wp-content/plugins/elementor-pro/base/module-base.php(60): ElementorPro\Modules\GlobalWidget\Module->__construct()
#1 /Users/parley/Local Sites/create-a-blog-with-elementor/app/public/wp-content/plugins/elementor-pro/includes/modules-manager.php(48): ElementorPro\Base\Module_Base::instance()
#2 /Users/parley/Local Sites/create-a-blog-with-elementor/app/public/wp-content/plugins/elementor-pro/plugin.php(239): ElementorPro\Manager->__construct()
#3 /Users/parley/Local Sites/create-a-blog-with-elementor/app/public/wp-includes/class-wp-hook.php(308): ElementorPro\Plugin->elementor_init('')
#4 /Users/parley/Local Sites/create-a-blog-with-elementor/app/public/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array)
#5 /Users/parley/Local Sites/create-a-blog-with-elementor/app/public/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /Users/parley/Local Sites/create-a-blog-with-elementor/app/public/wp-content/plugins/elementor/includes/plugin.php(652): do_action('elementor/init')
#7 /Users/parley/Local Sites/create-a-blog-with-elementor/app/public/wp-includes/class-wp-hook.php(308): Elementor\Plugin->init('')
#8 /Users/parley/Local Sites/create-a-blog-with-elementor/app/public/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters(NULL, Array)
#9 /Users/parley/Local Sites/create-a-blog-with-elementor/app/public/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#10 /Users/parley/Local Sites/create-a-blog-with-elementor/app/public/wp-settings.php(617): do_action('init')
#11 /Users/parley/Local Sites/create-a-blog-with-elementor/app/public/wp-config.php(77): require_once('/Users/parley/L...')
#12 /Users/parley/Local Sites/create-a-blog-with-elementor/app/public/wp-load.php(50): require_once('/Users/parley/L...')
#13 /Users/parley/Local Sites/create-a-blog-with-elementor/app/public/wp-admin/admin.php(34): require_once('/Users/parley/L...')
#14 /Users/parley/Local Sites/create-a-blog-with-elementor/app/public/wp-admin/plugins.php(10): require_once('/Users/parley/L...')
#15 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-11-02 20:38:32 [notice X 1][/Users/parley/Local Sites/create-a-blog-with-elementor/app/public/wp-content/plugins/elementor/modules/performance-lab/module.php::31] Required parameter $url follows optional parameter $size [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 2 of 2JS: 2022-11-02 20:39:27 [error X 1][http://localhost:10038/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.9.0-dev4:3:1061601] Cannot read properties of null (reading \'getBoundingClientRect\') 
JS: 2022-11-02 20:40:26 [error X 5][http://localhost:10038/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:31703] Cannot read properties of undefined (reading \'tools\') 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Incompatible

== Elementor Pro - Compatibility Tag ==
	

				
````

</details>

**Comment by nicholaszein** [comment]
Hello @CharfishCharlie!

Thank you for submitting your report! 🙏 

Can you please confirm you have added excerpts to posts? The Post Excerpt widget only show excerpts if they are added to the post:

![image](https://user-images.githubusercontent.com/113993460/199762304-75d65d80-21ef-4940-8215-8b1bc8279e7a.png)

**Comment by nicholaszein** [comment]
Yes, you can use the Dynamic Tag instead:

1. Add a Text or Heading element;
2. If it's a heading, change the HTML tag to something other than H1 or H2;
![image](https://user-images.githubusercontent.com/113993460/199774438-06efb5a0-125e-40fa-a97e-01eea8f3684f.png)

3. Click on the Dynamic icon and select "Post Excerpt";
![image](https://user-images.githubusercontent.com/113993460/199774574-5543a31e-abf6-42ae-80da-9ba7f1ad4551.png)

4. Click on the Wrench 🔧 icon and set your desired word length.
![image](https://user-images.githubusercontent.com/113993460/199774860-938ec8d7-9314-4794-9024-d7e6d9a20e38.png)


This should show the excerpt if you have it, and fallback to the post content if no excerpt exists.

Let me know if this works for you @CharfishCharlie! 😉

**Comment by CharfishCharlie** [comment]
Thanks for the suggestion. That didn't work either. Here are some images of what I see:

![Screen Shot 2022-11-03 at 9 25 02 AM](https://user-images.githubusercontent.com/1186866/199777618-5fc6eba2-c7c7-494c-9dd3-ebfb161606a3.png)
![Screen Shot 2022-11-03 at 9 25 10 AM](https://user-images.githubusercontent.com/1186866/199777650-a13b6739-e013-4a5d-af04-b6f0c9d18989.png)

The template I'm editing:
![Screen Shot 2022-11-03 at 9 25 53 AM](https://user-images.githubusercontent.com/1186866/199777781-3c0618e9-1232-4603-8892-55264cb07f31.png)

After saving and previewing:
![Screen Shot 2022-11-03 at 9 25 41 AM](https://user-images.githubusercontent.com/1186866/199777867-e281056d-2070-4a72-bfc1-84a6014c1dc7.png)

No excerpt shows up.

I'm proposing there should be something like this from the regular Posts widget, where you can bring in SOME content but not ALL content without the need to create/edit manual excerpts for potentially hundreds of posts:
![Screen Shot 2022-11-03 at 9 28 30 AM](https://user-images.githubusercontent.com/1186866/199778178-34f92a53-9bd0-498b-ad68-06fbd9a89710.png)

**Comment by syedwasiqbukhari** [comment]
Yes that's an issue I am facing as well

when I first bought Elementor Pro I was only working with sections, and i started using loop grid in sections, it was working fine there, even post excerpts. But when I turned on Flexbox containers, post excerpts vanished, and now when i try to turn off containers, it says you can't use loop grid without containers, strange !!!

When we manually insert excerpts in the post using a plugin or default excerpt section it than shows up there but it does not automatically fetch excerpt.

**Comment by CharfishCharlie** [comment]
Hi there. Any further input on this issue?

**Comment by donsony** [comment]
I can confirm that the issue exists in 8.1 as well.

If a excerpt is manually provided, it is displaying, but auto excerpts are not working

**Comment by nicholaszein** [comment]
Thank you for the feedback @CharfishCharlie!

I'm investigating this further and will get back to you with my findings!

**Comment by awebguy99** [comment]
+1 The way it should work (and works in the previous Posts widget) is, display the Manual Excerpt if it has any text.
If the Manual Excerpt is empty, then display the X number of characters from the beginning of the post content.

**Comment by gbaumbach** [comment]
+1 wanted to eliminate a plugin by leveraging this new built-in feature but without a way to excerpt post content, I can't use. The alternate method did not work for me either.

Post except should fallback to post content like the widget and *display* in the builder so we can actually style it :)

**Comment by vgstef** [comment]
Same problem here. I hope this will get fixed rapidly, because it screwed all my designs (show full contact instead of just a few words...).

**Comment by klaasgroenewold** [comment]
Please, fix this problem. :)

**Comment by vgstef** [comment]
Have you tried to add a Text widget. And in the dynamic tag, choose "Post summary" ?

**Comment by donsony** [comment]
> Have you tried to add a Text widget. And in the dynamic tag, choose "Post summary" ?

Tried this also, has the same issue. I believe the loop builder somehow affects the excerpt function.

This issue somehow makes the loop builder ineffective in most cases as auto excerpts are not being displayed. Please give top priority to fixing this.

**Comment by mchouvel** [comment]
Hi @nicholaszein, still no news about this issue ? Crazy to advertise so much this new widget with such an issue, it's not usable right now...

**Comment by klaasgroenewold** [comment]
Hey :) Post summary does indeed work for me! However, I have switched to Ele Custom Skin. Works even better than the Elementor loop and I don't face any issues with it.

**Comment by mchouvel** [comment]
> Hey :) Post summary does indeed work for me! However, I have switched to Ele Custom Skin. Works even better than the Elementor loop and I don't face any issues with it.

Thanks for your answer, but on my side I don't even see post summary in the dynamic tags of the text widget...
<img width="296" alt="Capture d’écran 2022-12-27 à 22 38 03" src="https://user-images.githubusercontent.com/20781791/209724862-f8981c4c-5a10-4536-babe-0e466c9e8eb8.png">

**Comment by ChrisWeiser** [comment]
> Hey :) Post summary does indeed work for me! However, I have switched to Ele Custom Skin. Works even better than the Elementor loop and I don't face any issues with it.

Would you please post screenshots of your setup? "Ele custom skin" sounds like something outside of the Post Loop, and I too am unfamiliar with "Post Summary."

**Comment by awakekat** [comment]
Using dynamic text with post excerpt also did not work for me in the Loop Grid...  I also would very much like if the manual excerpt is empty to force Elementor to grab the content like the Post Widget. We definitely do need more customization for the excerpt in the Loop grid. Some people do not want to take the extra time to write a specific excerpt for a blog post, the just want the first # of words as a teaser. Can we please get the content fall back with # of words if there is no manual excerpt.

**Comment by douvan** [comment]
+1 The expected behaviour of the Post Excerpt, is to display the Manual Excerpt if it has any text.
If the Manual Excerpt is empty, show the X number of characters from the beginning of the post content.
The new Elementor Loop Grid has limited value to a website with hundreds of posts with empty Manual Excerpts. Ele Custome Skin or Croco Elements can do this.  Third-party plugins wouldn't be needed if  Elementor Loop Grid worked properly.

**Comment by nadworks** [comment]
+1
No excerpt showing, which makes the Loop Builder useless indeed. We have hundreds of posts, so manually moving content into the custom except field is not feasible + extra work for staff when creating new post. It should work like the "Posts" widget, which pulls x-many words from the content if no custom excerpt is in place. **How come this has not been tested properly?**
Also: _post summary_ doesn't exist as dynamic field. Neither does _post content_.

**Comment by AurelienWebnation** [comment]
Same problem for me.

**Comment by Michaelgimii** [comment]
For me post Excerpt is displaying fine, but Excerpt Length is not working.

**Comment by Enno707** [comment]
Same problem also for me. Would be nice if the standard the_excerpt() functionality would be kept inside the loop builder. Plugins that hook into that function would then also be working again.

**Comment by reuhno** [comment]
Is there a filter to apply for quick fix ?

Same probleme here. Manual excerpt is working, but if it's empty... nothing !

Elementor Team, please do something

**Comment by patrykkozuch** [comment]
Made a simple workaround for this - simple `Better Post Excerpt` dynamic tag, which basically is the same as original `Post Excerpt` tag, but using `get_the_excerpt()` Wordpress function. Also added `Better Post Excerpt` widget, which initially uses this tag to display excerpt

[Link to the plugin](https://github.com/patrykkozuch/BetterElementorExcerpt)

**Comment by patrykkozuch** [comment]
Made also a fix for post excerpt default dynamic tag, which preserves current behaviour, but allows users to disable it.

@nicholaszein 

```php
<?php
namespace ElementorPro\Modules\DynamicTags\Tags;

use Elementor\Controls_Manager;
use ElementorPro\Core\Utils;
use ElementorPro\Modules\DynamicTags\Tags\Base\Tag;
use ElementorPro\Modules\DynamicTags\Module;

if ( ! defined( 'ABSPATH' ) ) {
	exit; // Exit if accessed directly
}

class Post_Excerpt extends Tag {
	public function get_name() {
		return 'post-excerpt';
	}

	public function get_title() {
		return esc_html__( 'Post Excerpt', 'elementor-pro' );
	}

	public function get_group() {
		return Module::POST_GROUP;
	}

	protected function register_controls() {

		$this->add_control(
			'max_length',
			[
				'label' => esc_html__( 'Excerpt Length', 'elementor-pro' ),
				'type' => Controls_Manager::NUMBER,
			]
		);

        $this->add_control(
            'only_manual',
            [
                'label' => esc_html__( 'Use only manual excerpt', 'elementor-pro'),
                'type' => Controls_Manager::SWITCHER,
                'default' => "yes"
            ]
        );
	}

	public function get_categories() {
		return [ Module::TEXT_CATEGORY ];
	}

    public function render() {
        // Allow only a real `post_excerpt` and not the trimmed `post_content` from the `get_the_excerpt` filter
        $post = get_post();

        if ( ! $post ) {
            return;
        }

        $settings = $this->get_settings_for_display();
        $max_length = (int) $settings['max_length'];
        $only_manual = $settings['only_manual'];

        if($only_manual == 'yes')
        {
            if ( empty($post->post_excerpt) )
            {
                return;
            }

            $excerpt = $post->post_excerpt;
        } else {

            $excerpt = get_the_excerpt();
        }

        $excerpt = Utils::trim_words( $excerpt, $max_length ) . '...';

        echo wp_kses_post( $excerpt );
    }
}
```

**Comment by dobsondobrev** [comment]
Hi patrykkozuch, I've been trying to find a solution to the problem of the Elementor Post Excerpt not working for a long time and thanks to BetterElementorExcerpt, the Excerpt now works, but unfortunately in Cyrillic the character count limitation doesn't work. Can you fix it? 
Thank you.

**Comment by rmcdsmith** [comment]
Before Elementor released their inbuilt loop builder I used Dudsters 'Elementor Custom Skin' plugin (Pro or Free, both do the trick). It adds a dynamic tag for Post Summary which does exactly as you all have proposed. It will show the Excerpt if one is set, otherwise it will show the first X number of words (which you can easily change to whatever number you like):

![image](https://user-images.githubusercontent.com/3713199/220524989-1cf81d39-4972-44ba-a8c1-dbb3efb97dd1.png)

![image](https://user-images.githubusercontent.com/3713199/220525025-ef7072f9-64b5-47dc-8fee-96cdeb5b040a.png)

Even though I rarely use this plugin for Loops anymore, I either use Elementor's new native Loops or JetEngine's Listings instead, I still find myself installing it just for this purpose when I need to (or until Elementor finally start supporting the Post Summary tag)

I'm sure someone cleverer than me could extract whatever the plugin is doing and create something we could use in the theme's functions.php to save another plugin install for such a menial task.

Hope this helps someone out in the meantime!

P.S. Long time GitHub voyeur, first time poster.

**Comment by sunnyt7** [comment]
@rmcdsmith I've re-installed Ele Custom Skin for this exact purpose as well.

Can't believe this is still a bug!

**Comment by emunot** [comment]
I can't believe this issue hasn't been addressed yet. This makes the loop feature useless for me.

**Comment by maziarom** [comment]
> Made a simple workaround for this - simple `Better Post Excerpt` dynamic tag, which basically is the same as original `Post Excerpt` tag, but using `get_the_excerpt()` Wordpress function. Also added `Better Post Excerpt` widget, which initially uses this tag to display excerpt
> 
> [Link to the plugin](https://github.com/patrykkozuch/BetterElementorExcerpt)


Thank you so much @patrykkozuch !!!

Your plugin works perfectly until, hopefully, Elementor fixes this issue.

**Comment by patrykkozuch** [comment]
@maziarom you don't even need to touch PHP code, after entering dynamic tag, just click on it and there will be two options - one for setting excerpt lenght, second for changing if WP should generate its own excerpt when manual excerpt is empty

**Comment by patrykkozuch** [comment]
> Hi patrykkozuch, I've been trying to find a solution to the problem of the Elementor Post Excerpt not working for a long time and thanks to BetterElementorExcerpt, the Excerpt now works, but unfortunately in Cyrillic the character count limitation doesn't work. Can you fix it? 
> Thank you.

@dobsondobrev Can you specify the issue more? Which font do you use?

**Comment by dobsondobrev** [comment]
> > Hi patrykkozuch, I've been trying to find a solution to the problem of the Elementor Post Excerpt not working for a long time and thanks to BetterElementorExcerpt, the Excerpt now works, but unfortunately in Cyrillic the character count limitation doesn't work. Can you fix it?
> > Thank you.
> 
> @dobsondobrev Can you specify the issue more? Which font do you use?

Hello patrykkozuch, Thank you for your reply.  I'm sorry couldn't explain the problem to you the first time.  I made a video to make it easier to see the problem about writing to you.  When changing the length of number of words the plugin reacts strangely. The behavior is different for Cyrillic and Latin.

https://user-images.githubusercontent.com/125753613/221812596-cdc745f0-4629-479c-8669-fce88830c8b4.mp4

https://user-images.githubusercontent.com/125753613/221814154-f0fff87f-251d-4565-9082-d3a48ffdf2c4.mp4

Thank you again.

**Comment by zerfl** [comment]
A couple of months later and automatic excerpts are still not working in the loop builder. 

@nicholaszein Is the team actively working on this or at least aware of the issue?

**Comment by nadworks** [comment]
I'd also like to know if there's going to be a built-in fix for this important missing ingredient yet. I'm about to move an entire magazine site to an Elementor theme and am relying on this to be working since we cannot manually add 400+ excerpts only because of the loop builder's limitations.
Appreciate that this is still an experimental feature. But a hint regarding a fix (at least a heads-up **that** you're working on it), would be worth getting.

**Comment by aerodinomx** [comment]
This is a big bug problem, hope it gets fixed soon.

**Comment by GiuMagnani** [comment]
Still nothing? The issue should have been solved by now

**Comment by dis-mal-skull** [comment]
Hi i write this little code, https://github.com/dis-mal-skull/Custom-Post-Excerpt-for-Elementor-Pro this works fine for me, because i have the same issue.

**Comment by awebguy99** [comment]
@nicholaszein can you please comment on this issue? It's major issue in the CORE part of the grid loop. This was opened in Nov 2022 and nothing from Elementor.

**Comment by nicholaszein** [comment]
Hello.

Please be patient. :wink:

**Comment by awebguy99** [comment]
> Hello.
> 
> Please be patient. 😉

Thanks for replying. Can you please give us a very approximate idea when this will be fixed? I have a number of projects starting soon. I'd love to use the new loop but this issue is a show stopper. Thank you. :)

**Comment by WebRat45** [comment]
+1, any idea on when this will be fixed?

**Comment by KoshaK0** [comment]
Just some clarification, that I think everyone missunderstand. This is not the issue or bug. This is Feature Request.
By default Content Widget showing the Whole content - that always was the case. Post Excerpt Widget was always showing the Excerpt if it's added to the Post. So by introducing the Loop Grid, Elementor did not added the functionality to Generate Excerpt from the post, as you can see this in other add-ons like Ele Custom skin. So - It's works as designed. It's not a bug.
It's does not need to be fixed, but need to add Excerp Generation functionality to the Post Widget or To Excerpt Widget and to Dynamic Option for Post Excert.

**Comment by KoshaK0** [comment]
> > ... It's works as designed. It's not a bug. It's does not need to be fixed, but need to add Excerp Generation functionality to the Post Widget or To Excerpt Widget and to Dynamic Option for Post Excert.
> 
> Pardon my ignorance, but isn't auto-excerpts part of core WP functionality? So, rather than "adding" that functionality, Elementor should just use what's already there, not suppress it. It was **that** complex, other less sophisticated page builders would equally ignore it. Only they don't.

I never seen the "auto-excerpts" as art of WP Functionality. It's might be the Functionality of Gutenberg Widgets, which is also included into the CORE as Add-onn so any functionality in Gutenberg is not the same as the WP CORE functionality. If that makes sence... So Any Elementor Functionality does not related to the "WP CORE" Functionality. Elementor never had Auto-Excerpt at all. I think you see my point.

**Comment by Michaelgimii** [comment]
Since this issue is planned to be resolved for version 3.13, I wanted to point out a very small issue in the Excerpt. In the Excerpt at the end of the last word, there is no ... (three points) Displaying three dots at the end of the abstract makes users understand that this is an abstract and the text continues.

**Comment by francis150** [comment]
> Since this issue is planned to be resolved for version 3.13, I wanted to point out a very small issue in the Excerpt. In the Excerpt at the end of the last word, there is no ... (three points) Displaying three dots at the end of the abstract makes users understand that this is an abstract and the text continues.

![image](https://user-images.githubusercontent.com/56007181/232023164-9ad7149e-65b4-435a-ae22-7c16855eac50.png)

**Comment by pixelloop** [comment]
> Since this issue is planned to be resolved for version 3.13, I wanted to point out a very small issue in the Excerpt. In the Excerpt at the end of the last word, there is no ... (three points) Displaying three dots at the end of the abstract makes users understand that this is an abstract and the text continues.

I think that's to be expected with the current approach as it only allows manual excerpts, these should be complete statements and don't need the ellipses. 

I definitely agree that this should be added if/when the default WordPress excerpt functionality works again and it's forced to automatically get the first 50 words instead of using the manual excerpt though.

**Comment by nicholaszein** [comment]
Hello everyone!

This issue is going to be solved with Elementor Pro version 3.13. We are almost there! 😉 🙌

**Comment by pixelloop** [comment]
> Hello everyone!
> 
> This issue is going to be solved with Elementor Pro version 3.13. We are almost there! 😉 🙌

Great news, is there an ETA for that version to be released?

**Comment by awebguy99** [comment]
> Hello everyone!
> 
> This issue is going to be solved with Elementor Pro version 3.13. We are almost there! 😉 🙌

Great news! Thanks for the update. :)

**Comment by gsqrt2** [comment]
Using the post summary dynamic value did not work for me. While waiting for the fix, a quick workaround is to just register a shortcode returning the the excerpt:

`add_shortcode('my_post_excerpt', function(){
	the_excerpt();
});`

and then use the shortcode dynamic value to display it in any widget (heading, text editor etc).

**Comment by nicholaszein** [comment]
Hey everyone!

We are happy to announce that this new feature is now live in the latest v3.13-Beta release!

We would love to see you try this new feature and give your feedback in the [Beta Announcement](https://github.com/orgs/elementor/discussions/22119)! 💪

Please try it out and let us know!

https://github.com/orgs/elementor/discussions/22119

Kind regards

**Comment by pixelloop** [comment]
@nicholaszein 

Thanks for the update. Not too long to wait, then! That's good.

**Comment by diehendrick** [comment]
I have been waiting for this issue to solve it, and still blocker for me.  And tested elementor  3.13.0-dev3, still not solved. 
and also post title, every time I enable link on the title the font will be changed. I have no idea.  loop builder is great. but please should be priority. 

I tested using hello elementor.  @nicholaszein  

https://user-images.githubusercontent.com/1661591/235126965-c238f2b7-1f10-4efb-949f-53b434cd5ef8.mp4

**Comment by 79ho3ein** [comment]
Hi @diehendrick 

At the 27th second of your video, it seems that the new "Apply to post content" option is missing for you.

![1](https://user-images.githubusercontent.com/77163248/235134661-35eeaace-c982-44e3-a0bc-5456f2b6ff1d.png)

**Comment by diehendrick** [comment]
> Hi @diehendrick
> 
> At the 27th second of your video, it seems that the new "Apply to post content" option is missing for you.
> 
> ![1](https://user-images.githubusercontent.com/77163248/235134661-35eeaace-c982-44e3-a0bc-5456f2b6ff1d.png)

Hi, @79ho3ein 

on post excerpt element, heading or text editor? I double check there are not showing "apply to post content" toggle option

**Comment by zdmanning** [comment]
> Hey everyone!
> 
> We are happy to announce that this new feature is now live in the latest v3.13-Beta release!
> 
> We would love to see you try this new feature and give your feedback in the [Beta Announcement](https://github.com/orgs/elementor/discussions/22119)! 💪
> 
> Please try it out and let us know!
> 
> https://github.com/orgs/elementor/discussions/22119
> 
> Kind regards

As diehendrick stated and demonstrated, it does not work in the beta release. The same was true for me.

**Comment by zdmanning** [comment]
Wow, strong Elementor… what a joke…

**Comment by AdictiveDesign** [comment]
I am very confused as while i build my own website www.adictivedesign.de there where no issues with the excerpt but now, while building a clients website, i have the same issue as yall have.
I wasnt able to figure out, what I did differently on my website but i simply used the "post grid" element and adjusted the settings in this element and not with a plugin, css or whatsoever..

**Comment by nicholaszein** [comment]
Hello everyone!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **v3.13.0**! 🥳 

You can now use the Post Excerpt widget and Dynamic Tag, and switch on the option to `Apply to post content` to include part of your post content in case you don't have a post excerpt.

![image](https://user-images.githubusercontent.com/113993460/236926863-e290e71c-f2e3-43c3-ae48-aa24bdcc94b7.png)
----

✅ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
